interface/modules/zend_modules/module/Carecoordination/view/layout/encountermanager.phtml

```diff
@@ -28,13 +28,13 @@
 		->prependFile($this->basePath() . '/js/autosuggest/autosuggest.js')
         ->prependFile($this->basePath() . '/js/application/sendTo.js')
         ->prependFile($this->basePath() . '/js/carecoordination/encounterManager.js')
-        ->prependFile($this->basePath() . '/js/carecoordination/jquery-ui.custom.js')
-        ->prependFile($this->basePath() . '/js/carecoordination/jquery.ui.datepicker.js')
-        ->prependFile($this->basePath() . '/js/carecoordination/jquery.ui.core.js')
+        ->prependFile($this->basePath() . '/js/lib/jquery-ui.custom.js')
+        ->prependFile($this->basePath() . '/js/lib/jquery.ui.datepicker.js')
```

Comment:
Testing: We should at least point to /public/assets in this file so we can get rid of the excess files.